tests/acceptance/features/coreApiSharees/sharees.feature

```diff
@@ -273,8 +272,7 @@ Feature: sharees
 
 
   Scenario Outline: search for sharees with long name when search min length is set to 2
-    Given the administrator has updated system config key "user.search_min_length" with value "2"
-    And user "sharee2" has been created with default attributes and without skeleton files
+    Given user "sharee2" has been created with default attributes and without skeleton files
```

Comment:
What is the default "search min length" on oCIS?

We need to put a comment that this scenario is only expected to work if the system-under-test already supports providing only 2 characters in the search string.

Reply:
The default `search_min_length` is 2 if I am not wrong https://gist.github.com/grgprarup/43d9df31d469ca0cf146696ae6c64c39#:~:text=%22search_min_length%22%3A%202%2C